spec/02-integration/05-proxy/09-balancer_spec.lua

```diff
@@ -371,194 +454,24 @@ local localhosts = {
 }
 
 
-for mode, localhost in pairs(localhosts) do
+dao_helpers.for_each_dao(function(kong_config)
```

Comment:
Here however, we really should rely on the proper utility. Combined with `get_db_utils()`, we get:
```lua
for _, strategy in helpers.each_strategy(strategy?) do
describe("<component> [#" .. strategy .. "]", function()

  setup(function()
    local bp, db, dao = helpers.get_db_utils(strategy) -- also runs migrations for us

    assert(helpers.start_kong({
      database   = strategy,
      nginx_conf = "spec/fixtures/custom_nginx.template",
    }))
  end)
end)
end
```